app/views/content/how-to-apply/_questions.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="category__questions__container">
+
+  <section class="container category__questions">
+
+    <div class="category__question">
+      <h2>Want to chat with an adviser?</h2>
+
+      <p>
+        Get free, one-to-one support with your application from an adviser with years of teaching experience.
```

Comment:
```suggestion
        Get free, one-to-one support with your application from an adviser with years of teaching experience.
```
Should we emphasise `before you apply`?

Reply:
do we need 'before you apply' if we're saying 'with your application'?

Reply:
let's leave as is then!